workflow/rules/commons/05_refcon.smk

```diff
@@ -17,7 +17,14 @@ import pandas
 
 localrules: refcon_run_dump_manifest, refcon_cache_manifests
 
-DIR_REFCON = pathlib.Path(config['reference_container_folder'])
+
+USE_REFCON = config['USE_REFCON']
```

Comment:
as before: should be more explicit, i.e. "USE_REFERENCE_CONTAINER" and then "USE_REFCON" only as alias

Reply:
I changed the names of the the variables to be more explicit and also used lower case for config items